doc/nrf/libraries/networking/multicell_location.rst

```diff
@@ -9,7 +9,7 @@ Multicell location
 
 .. note::
 
-   This library is deprecated since v2.3.0 and relevant functionality is available through the :ref:`lib_location` library.
+   This library is deprecated after |NCS| v2.2.0 and relevant functionality is available through the :ref:`lib_location` library.
```

Comment:
```suggestion
   This library has been marked as :ref:`deprecated <software_maturity>` after the release of the |NCS| v2.2.0 and relevant functionality is available through the :ref:`lib_location` library.
```
The actual link should point to the API deprecation section, so you might want to add the tag for that section on the software maturity page and link exactly there.